src/modules/dashboard/components/ColonyHome/ColonyMeta/ColonyInvite.tsx

```diff
@@ -1,46 +0,0 @@
-import React from 'react';
-import { defineMessages, FormattedMessage } from 'react-intl';
-
-import ClipboardCopy from '~core/ClipboardCopy';
-
-import styles from './ColonyInvite.css';
-
-const MSG = defineMessages({
-  inviteLabel: {
-    id: 'dashboard.ColonyHome.ColonyMeta.ColonyInvite.inviteLabel',
-    defaultMessage: 'Invite your community to join',
-  },
-  copyLabel: {
-    id: 'dashboard.ColonyHome.ColonyMeta.ColonyInvite.copyLabel',
-    defaultMessage: `{valueIsCopied, select,
-      true {copied}
-      false {copy}
-    }`,
-  },
-});
-
-interface Props {
-  colonyName: string;
-}
-
-const displayName = 'dashboard.ColonyHome.ColonyMeta.ColonyInvite';
-
-const ColonyInvite = ({ colonyName }: Props) => {
```

Comment:
Maybe the colony invite logic shouldn't have been deleted.

Then again, leave it like this, I'll re-add what I need in my PR.